packages/react-native-sdk/src/components/CallControlsView.tsx

```diff
@@ -4,25 +4,52 @@ import { useCallControls } from '../hooks/useCallControls';
 import { CameraSwitch, Chat, PhoneDown, Reaction } from '../icons';
 import { CallControlsButton } from './CallControlsButton';
 import { theme } from '../theme';
-import { CallingState, OwnCapability } from '@stream-io/video-client';
-import {
-  Restricted,
-  useCall,
-  useCallCallingState,
-} from '@stream-io/video-react-bindings';
+import { OwnCapability } from '@stream-io/video-client';
+import { Restricted } from '@stream-io/video-react-bindings';
 import { ReactionModal } from './ReactionsModal';
 import { ToggleAudioButton } from './ToggleAudioButton';
 import { ToggleVideoButton } from './ToggleVideoButton';
 import { A11yButtons, A11yComponents } from '../constants/A11yLabels';
 import { Z_INDEX } from '../constants';
 
+/**
+ * The props for the Chat Button in the Call Control View.
+ */
 type ChatButtonType = {
+  /**
+   * Handler to be called when the chat button is pressed.
+   * @returns void
+   */
   onPressHandler: () => void;
+  /**
+   * The unread message indicator to be displayed above on the Chat button.
+   */
   unreadBadgeCountIndicator?: number;
 };
 
+/**
+ * The props for the Hang up call button in the Call Control View.
+ */
+type HangUpCallButtonType = {
+  /**
+   * Handler to be called when the hang up button is pressed.
+   * @returns void
+   */
+  onPressHandler: () => void;
+};
```

Comment:
I think we can make it simpler for the integrator just by using the below type instead

`onHangupButtonPress`

Reply:
We decided on this pattern so that in future, we can have more props, like styles, etc., that might be needed for the button. Eg, We have one for `chatButton` as an unreadIndicator.

CC: @vanGalilea

Reply:
The button can be moved to a component too, to make the `CallControlsView` look simpler/cleaner, but that can be done later.